yt_dlp/extractor/playsuisse.py

```diff
@@ -0,0 +1,121 @@
+# coding: utf-8
+import json
+
+from .common import InfoExtractor
+from ..utils import int_or_none
+
+
+class PlaySuisseIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?playsuisse\.ch/watch/(?P<id>[0-9]+)'
+    _TESTS = [
+        {
+            'url': 'https://www.playsuisse.ch/watch/763211/0',
+            'md5': '0d716b7a16c3e6ab784ef817ee9a20c1',
+            'info_dict': {
+                'id': '763211',
+                'ext': 'mp4',
+                'title': 'Knochen',
+                'description': 'md5:8ea7a8076ba000cd9e8bc132fd0afdd8'
+            }
+        },
+        {
+            'url': 'https://www.playsuisse.ch/watch/808675/0',
+            'md5': '7c59b60aadd84b3e36d46dea01125442',
+            'info_dict': {
+                'id': '808675',
+                'ext': 'mp4',
+                'title': 'Der Läufer',
+                'description': 'md5:9f61265c7e6dcc3e046137a792b275fd'
+            }
+        },
+        {
+            'url': 'https://www.playsuisse.ch/watch/817193/0',
+            'md5': 'eff6791c38784543f6d87a58bfe5de15',
+            'info_dict': {
+                'id': '817193',
+                'ext': 'mp4',
+                'title': 'Die Einweihungsparty',
+                'series': 'Nr. 47',
+                'season_number': 1,
+                'episode': 'Die Einweihungsparty',
+                'episode_number': 1,
+                'description': 'md5:91ebf04d3a42cb3ab70666acf750a930'
+            }
+        }
+    ]
+
```

Comment:
```suggestion
_NETRC_MACHINE = 'playsuisse'

```
We might want to add this to make it work with netrc.

Reply:
The extractor hasn't added login support. So _NETRC_MACHINE should not be added

Reply:
You're right, I've only read the description...